maintenance/guess_league_archetypes.py

```diff
@@ -3,8 +3,10 @@
 HOURLY = True
 
 def run() -> None:
+    archetype.clear_old_predictions()
     decks = deck.load_decks('archetype_id is null')
     deck.calculate_similar_decks(decks)
     for d in decks:
-        if d.similar_decks:
-            archetype.assign(d.id, d.similar_decks[0].archetype_id, False)
+        if d.similar_decks and d.similar_decks[0].archetype_id is not None:
```

Comment:
Should we spider down until we find a similar deck that *does* have an archetype rather than just punting if top match doesn't have one? Top match could be another recent deck that hasn't been categorized and they could both be copies of a categorized deck. If we do this we'd probably want a minimum threshold but I think that's covered by calculate_similar_decks.

Reply:
Done. Since we're showing a confidence score, no real harm in putting in the best we can find and if the confidence score ends up being 10% or whatever then you can ignore the predicted archetype.